src/components/SlateEditor/plugins/listText/actions.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2021-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ *
+ */
+
+export function getCurrentListText(value) {
+  if (!value.selection.start.key) return null;
+  const { startBlock } = value;
+  return startBlock && startBlock.type === 'list-text' ? startBlock : null;
+}
+
+export function onEnter(evt, editor, next) {
+  const currentListText = getCurrentListText(editor.value);
+  if (!currentListText || !evt.shiftKey) {
+    return next();
+  }
+  evt.preventDefault();
+
+  return editor
+    .delete()
+    .insertBlock('br')
```

Comment:
Skal det være to br her?

Reply:
Jeg synes det virket mest naturlig. Da får man en tom linje mellom den gamle tekstblokken og den nye man oppretter.

Droppet også bruken av 'paragraph' siden det introduserte nye problemer med forskjell i marginer og endring av oppførselen til shift+enter.